backend/resources/reviews.py

```diff
@@ -0,0 +1,51 @@
+from flask_restful import Resource, reqparse, abort
+from flask import g
+from model.users import UsersModel, auth, Roles
+from model.library import LibraryModel, LibraryType, State
+from model.books import BooksModel
+from model.reviews import ReviewsModel
+
+
+def parse_review():
+    parser = reqparse.RequestParser(bundle_errors=True)
+
+    parser.add_argument('isbn', required=True, type=int,
+                        help="ISBN of the book associated to the review. Can't be null.")
+    parser.add_argument('email', required=True, type=str,
+                        help="Email of the user associated to the review. Can't be null.")
+    parser.add_argument('score', required=True, type=int,
+                        help="Integer score ranging from 1 to 5 which indicates book score. Can't be null.")
+    parser.add_argument('review', required=False, type=str,
+                        help="Review message the user has posted with the score. Can be null.")
+
+    return parser.parse_args()
+
+
+class Reviews(Resource):
+
+    @auth.login_required
+    def post(self):
+        data = parse_review()
+        user = UsersModel.find_by_email(data['email'])
+        if g.user != user:
+            return {"message": "Invalid user to remove, can only be yourself"}, 401
+
+        if not user:
+            return {"message": f"User with ['email': {data['email']}] Not Found"}, 404
```

Comment:
Seria millor comprovar si el user es None abans, perque sino mai entrara degut a que g.user mai es None.
```suggestion python
        if not user:
            return {"message": f"User with ['email': {data['email']}] Not Found"}, 404

        if g.user != user:
            return {"message": "Invalid user to remove, can only be yourself"}, 401
```
Altre opcio es no comprovar-ho.
```suggestion python
        if g.user != user:
            return {"message": "Invalid user to remove, can only be yourself"}, 401
```